src/data/setoid/basic.lean

```diff
@@ -374,14 +374,16 @@ begin
   exact quotient.eq',
 end
 
+@[simp]
+lemma top_rel_eq_triv_rel : (⊤ : α → α → Prop) = λ _ _, true := by {refl}
+
```

Comment:
I added this lemma as it seemed useful (and I used it twice).

Reply:
I don't think we want this lemma, `top_apply` is a better version that already exists.

Reply:
Fixed